src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -151,6 +151,13 @@ protected final void setExpr(final Expression<? extends T> expr) {
 		this.expr = expr;
 	}
 	
+	/**
+	 * @return The expression this condition checks the property of
+	 */
+	public final Expression<? extends T> getExpr() {
```

Comment:
```suggestion
	public Expression<? extends T> getExpr() {
```

Reply:
Shouldn't it be not override-able?

Reply:
I think methods should only be made final if they have a good reason you should not be able to override them. 
Also, the method should be protected.